roles/neutron-common/templates/usr/local/bin/neutron-restart-all

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+restart neutron-server
+restart neutron-linuxbridge-agent
```

Comment:
Yeah that's desired, as it gets them back on the rabbit bus and the service migration works.